app/controllers/admin_controller.rb

```diff
@@ -1,6 +1,12 @@
 class AdminController < ApplicationController
   before_filter :require_user, only: %i(spam spam_revisions mark_comment_spam publish_comment)
 
+  params.require(:comment).permit(:pid, :nid, :uid, :aid, :subject, :hostname, :comment, :status, :format, :thread, :timestamp)
+  params.require(:drupal_user).permit(:title, :body, :name, :pass, :mail, :m    ode, :sort, :threshold, :theme, :signature, :signature_format, :created, :ac    cess, :login, :status, :timezone, :language, :picture, :init, :data, :timezo    ne_id, :timezone_name)
+  params.require(:node).permit(:title, :uid, :status, :type, :vid, :cached_likes, :comment, :path, :slug, :views)
+  params.require(:revision).permit(:title, :body, :nid, :uid, :teaser, :log, :timestamp, :format)
+  params.require(:user).permit(:username, :email, :password, :password_confirmation, :openid_identifier, :key, :photo, :photo_file_name, :bio, :status)
+
```

Comment:
How are you using these parameters? I am thinking you'll make them private methods and plug them into where they are needed?

Reply:
Yeah @siaw23, we can do that too. I will do that way. Thanks!!